### DISCUSSION TITLE
Unable to connect for light MSL120DR

### DISCUSSION THREAD

**Discussion by jmsblah** [krahabb/meross_lan#140] [Q&A]
Hi!
I am new to Home Assistant and I am trying to setup Meross LAN to control some lights using Lutron Pico remotes.
I am not being able to add lights using Merross LAN, I am getting Unable to connect when I try to add a light manually. Only once one light appeared automatically to be added. Any advice?

**Comment by krahabb** [comment]
Hello @jmsblah ,
I'm sorry but I cannot understand the reference to Lutron remotes. meross_lan only works with Meross devices and I don't see the relation between the two.
Maybe you are using Lutron switches to turn on/off a Meross bulb (MSL120DR) ?
If that's the case keep in mind that the device (bulb) can only be managed and used when the Lutron turns it on like if it were a common 'dumb' wall switch. If you turn off the 'wall' switch the bulb is not powered so it cannot communicate with HA/meross_lan

I'm sorry if I didn't correctly figured out the issue
